stdlib/universe/distinct_test.flux

```diff
@@ -1,5 +1,6 @@
 package universe_test
```

Comment:
I noticed that this file was not correctly formatted when I merged that PR. For whatever reason, this passed CI. I'm just including this fix in this PR since it isn't a major change. This is just a formatting change.